### ISSUE TITLE
False positive for SC2034 with `read -i`

### ISSUE THREAD

**Issue by mapreri** [koalaman/shellcheck#1796]
#### For bugs
- Rule Id: `SC2034` ("unused variable") - https://github.com/koalaman/shellcheck/wiki/SC2034
- My shellcheck version: `0.7.0`
- [x] The rule's wiki page does not already cover this
- [x] I tried on shellcheck.net and verified that this is still a problem on the latest commit

#### Here's a snippet or screenshot that shows the problem:

```sh
#!/bin/bash

read -r -e -p "Are you sure? " -i l y
echo "$y"
```

#### Here's what shellcheck currently says:

```
In foo.sh line 3:
read -r -e -p "Are you sure? " -i l y
                                  ^-- SC2034: l appears unused. Verify use (or export if used externally).

For more information:
  https://www.shellcheck.net/wiki/SC2034 -- l appears unused. Verify use (or ...
```

#### Here's what I wanted or expected to see:

Nothing, since 'l' there is not a variable.
Quoting doesn't seem to help either.

**Comment by e-kwsm** [comment]
Seems to be fixed at some point (shellcheck 0.11.0).
